addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -193,8 +176,43 @@ function factory(dependencies) {
          *
          * @param {MouseEvent} ev
          */
-        onClickStartAMeetingButton(ev) {
-            this.createMeetingChannel();
+        async onClickStartAMeetingButton(ev) {
+            const meetingChannel = await this.messaging.models['mail.thread'].createGroupChat({
+                default_display_mode: 'video_full_screen',
+                partners_to: [this.messaging.currentPartner.id],
+            });
+            if (!this.exists()) {
+                return;
+            }
```

Comment:
discuss being deleted shouldn't be prevent from actually opening the channel and starting the call (in chat window if it must be)